### DISCUSSION TITLE
当无法正常打开 simpread.pro 的解决方案（加快 simpread.pro 系网站的加载）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#6590] [重要通知]
> 如果近期遇到打开简悦系官网较慢 simpread.pro 等无法加载的问题 ，请看下面的解决方案。

### 问题原因

因为  `staticfile` `bootcdn` CDN 出现问题导致。

### 解决方案

简悦全部服务的 CDN 已经迁移到 `s4.zstatic.net` 因此不会发生此类问题。

### 解决时间

2024年7月20日（含）已彻底解决此问题。

**Comment by Francklin** [comment]
staticfile 和bootcdn 有问题，推荐使用webcache.cn替代，支持cdnjs, npm 和github, 支持SRI。刚刚测试贵站官网，s4.zstatic.net 在湖北地区打不开。
![image](https://github.com/user-attachments/assets/939c059a-ff32-40c8-9b29-1e60a9e183f6)

**Reply by Kenshin** [reply]
嗨，谢谢提示，简悦全部官网均已去掉了 `staticfile` `bootcdn`

未免误导普通用户，我删除了你提到的某些词。 

至于你提到的问题，我在使用 `s4.zstatic.net` 已经做了测试，目前没有遇到你提到的情况。

未免以后再迁移，所以我打算使用自己的 CDN 但暂时还没有上线。

我的测试地址：

1. https://ping.chinaz.com/s4.zstatic.net

    <img width="1255" alt="image" src="https://github.com/user-attachments/assets/00f7f28b-25c8-4022-83ae-79cbe804a5be">

2. https://www.itdog.cn/ping/s4.zstatic.net

     <img width="1440" alt="image" src="https://github.com/user-attachments/assets/24596ae6-86e6-4034-b5f7-f976872baebc">
